lambda-resume-builder/src/Components/Assignments/Assignments.js

```diff
@@ -0,0 +1,24 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+
+const Assignments = () => {
+   
+        return (
+            <h1>This Is The Assignments Checklist Component</h1>
+        )
+    
+        
+    
+    
+    
+}
+
+
```

Comment:
### I don't know who added all of this extra white space, but it really needs to get taken out **ASAP**.